src/hooks/useImmediateEffect.ts

```diff
@@ -1,4 +1,6 @@
-import { useRef, useEffect, useCallback } from 'react';
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Is this intentional? Seems somewhat dangerous and easy to overlook. What would you say to just using `eslint-disable-next-line` where needed?